src/vm/codeman.cpp

```diff
@@ -6898,50 +6898,56 @@ BOOL ReadyToRunJitManager::JitCodeToMethodInfo(RangeSection * pRangeSection,
     // READYTORUN: FUTURE: Hot-cold spliting
 
     TADDR currentInstr = PCODEToPINSTR(currentPC);
+    TADDR imageBase = pRangeSection->LowAddress;
 
-    TADDR ImageBase = pRangeSection->LowAddress;
-
-    DWORD RelativePc = (DWORD)(currentInstr - ImageBase);
+    DWORD relativePc = (DWORD)(currentInstr - imageBase);
 
     Module * pModule = dac_cast<PTR_Module>(pRangeSection->pHeapListOrZapModule);
     ReadyToRunInfo * pInfo = pModule->GetReadyToRunInfo();
 
     COUNT_T nRuntimeFunctions = pInfo->m_nRuntimeFunctions;
     PTR_RUNTIME_FUNCTION pRuntimeFunctions = pInfo->m_pRuntimeFunctions;
 
-    int MethodIndex = NativeUnwindInfoLookupTable::LookupUnwindInfoForMethod(RelativePc,
+    int methodIndex = NativeUnwindInfoLookupTable::LookupUnwindInfoForMethod(relativePc,
```

Comment:
It would be nice to do the cosmetic changes in separate change, so that this is easy to port to servicing branch.

Also, the same code is in `NativeImageJitManager::JitCodeToMethodInfo` as well.

Reply:
Good point, will do.